### DISCUSSION TITLE
Switch spaces in .MAP file to tabs (\t) instead

### DISCUSSION THREAD

**Discussion by sukus21** [gbdev/rgbds#1012] [Discussions]
Opening the .map files generated by RGBLINK in a code editor can look off for some (me included). The cause of this is the indentation is done with spaces instead of tabs (\t). This would be a simple fix, and would personally benefit me greatly :)
If there's a specific reason it's like this, please enlighten me, I would love to know!

I have links to the relevant sections of code in link/output.c below here

https://github.com/gbdev/rgbds/blob/f3f2c2ca16ba3d624829bcece86c60f0e060c4ac/src/link/output.c#L387-L395

https://github.com/gbdev/rgbds/blob/f3f2c2ca16ba3d624829bcece86c60f0e060c4ac/src/link/output.c#L399-L407

https://github.com/gbdev/rgbds/blob/f3f2c2ca16ba3d624829bcece86c60f0e060c4ac/src/link/output.c#L414

https://github.com/gbdev/rgbds/blob/f3f2c2ca16ba3d624829bcece86c60f0e060c4ac/src/link/output.c#L418-L419

**Comment by ISSOtm** [comment]
The problem with tabs is that they have inconsistent width (2, 4, and 8 are the most common) and so tend to create unaligned columns.
What's wrong with spaces?

(Note: using tabs for *indentation* is fine, but not for alignment.)

**Reply by sukus21** [reply]
Ohhh gotcha, I wasn't thinking about alignment, just indentation. I see now how it's all nicely aligned, I'm shocked I never noticed that before.

I have just been thinking about it the wrong way, and regardless, it really isn't an issue to begin with. My bad ^^
